zproject/urls.py

```diff
@@ -365,8 +365,8 @@
     # users/me/alert_words -> zerver.views.alert_words
     url(r'^users/me/alert_words$', rest_dispatch,
         {'GET': 'zerver.views.alert_words.list_alert_words',
-         'POST': 'zerver.views.alert_words.set_alert_words',
-         'PUT': 'zerver.views.alert_words.add_alert_words',
+         'POST': 'zerver.views.alert_words.add_alert_words',
+         'PUT': 'zerver.views.alert_words.set_alert_words',
```

Comment:
I thought our idea was to just remove the `PUT` version and only have `POST` be `add_alert_words`.  We don't need to support `set_alert_words` as an interface anymore; we should just remove the code for it.

Reply:
Thanks for the feedback! That's what I initially thought :) But then I noticed that replacing the list of alert words with new values seems useful in tests (for example, [here](https://github.com/alenavolk/zulip/blob/b83fc18ec04a13aa7e90f00e1af9746242965f17/zerver/tests/test_alert_words.py#L33)) and got confused. I'll remove the PUT endpoint and change the test then.